test/feature/support/before_after.coffee

```diff
@@ -1,22 +1,21 @@
-portfinder = require 'portfinder'
-tmp = require 'tmp'
+Promise = require 'bluebird'
+portfinder = Promise.promisifyAll require 'portfinder'
```

Comment:
Yeah this could work, but would make it harder to change if I ever need another bluebird function or the bluebird `Promise` constructor.

I don't feel too strongly either way about it though